### PR TITLE
go fwd - support multiple ports, cleanup run_fwd and PortStats

### PR DESCRIPTION
**Pull request by dakshinai**
Signed-off-by: Dakshina Ilangovan <dakshina.ilangovan@intel.com>

- Support multiple ports in go fwd example
- remove hardcoded paths in run_fwd and allow the option to support multiple ports
- cleanup PortStats - remove dependency on FreePortStats()

### CODE REVIEW

File: lang/go/cndp.org/examples/fwd/fwd.json [hunk 1]
```diff
@@ -35,6 +35,13 @@
             "umem": "umem0",
             "region": 0,
             "description": "LAN 0 port"
+        },
+        "enp25s0f1:0": {
+            "pmd": "net_af_xdp",
+            "qid": 37,
+            "umem": "umem0",
+            "region": 1,
+            "description": "LAN 0 port"
```
Comment (KeithWiles) [inline lang/go/cndp.org/examples/fwd/fwd.json:44]: This should be 'LAN 1 port'

File: lang/go/cndp.org/examples/fwd/run_fwd [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
-sudo -E LD_LIBRARY_PATH=$project_path/usr/local/lib/x86_64-linux-gnu LD_PRELOAD=$project_path/usr/local/lib/x86_64-linux-gnu/libpmd_af_xdp.so ./fwd -c fwd.json -t rx -p enp25s0f0:0
 
-stty sane
+cmdstring="sudo -E LD_LIBRARY_PATH=$PROJECT_PATH/usr/local/lib/x86_64-linux-gnu \
+LD_PRELOAD=$PROJECT_PATH/usr/local/lib/x86_64-linux-gnu/libpmd_af_xdp.so \
+./fwd -c $config -t $test $lportsstr"
+
+eval $cmdstring
+
+stty sane
```
Comment (KeithWiles) [inline lang/go/cndp.org/examples/fwd/run_fwd:32]: No newline at end of file.

File: lang/go/cndp.org/examples/fwd/run_fwd [hunk 3]
```diff
@@ -1,8 +1,32 @@
...
+    lportsstr+="-p `echo $lport | xargs` "; 
+done
 
 go build
-sudo -E LD_LIBRARY_PATH=$project_path/usr/local/lib/x86_64-linux-gnu LD_PRELOAD=$project_path/usr/local/lib/x86_64-linux-gnu/libpmd_af_xdp.so ./fwd -c fwd.json -t rx -p enp25s0f0:0
 
-stty sane
+cmdstring="sudo -E LD_LIBRARY_PATH=$PROJECT_PATH/usr/local/lib/x86_64-linux-gnu \
+LD_PRELOAD=$PROJECT_PATH/usr/local/lib/x86_64-linux-gnu/libpmd_af_xdp.so \
+./fwd -c $config -t $test $lportsstr"
```
Comment (KeithWiles) [inline lang/go/cndp.org/examples/fwd/run_fwd:28]: Is this script run using sudo i.e., 'sudo ./run_fwd ...' or should you include sudo on this line to allow the user to do './run_fwd ...'? We can not expect the developer/user to be logged in as root to run CNDP applications.

**Review by KeithWiles — CHANGES_REQUESTED** [review]

File: lang/go/cndp.org/examples/fwd/run_fwd (hunk 3 above)
Comment (dakshinai) [reply lang/go/cndp.org/examples/fwd/run_fwd:28]: yes the command executes with sudo, starting line 26

File: lang/go/cndp.org/examples/fwd/run_fwd (hunk 2 above)
Comment (dakshinai) [reply lang/go/cndp.org/examples/fwd/run_fwd:32]: im assuming you are saying newline be added?

**Review by KeithWiles — APPROVED** [review]

**Pull request merged**